test/tests/tsdx-build-closure-compiler.test.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * @jest-environment node
+ */
+
+const shell = require('shelljs');
+const util = require('../fixtures/util');
+
+shell.config.silent = false;
+
+const grep = (pattern, file) => {
+  const output = shell.grep(pattern, file);
+  const grepOutput = output.stdout.replace(`\n`, ``);
+  return grepOutput.length > 0;
+};
+
+const stageName = 'stage-build-closure-compiler';
+
+describe('tsdx build with closure compiler default options', () => {
+  beforeAll(() => {
+    util.teardownStage(stageName);
+  });
+
+  it('should minify bundle with default options', () => {
+    util.setupStageWithFixture(stageName, 'build-withConfig');
+
+    let output = shell.exec(
+      'node ../dist/index.js build --env production --closureCompiler'
+    );
+    expect(output.code).toBe(0);
+
+    expect(shell.test('-f', 'dist/index.js')).toBeTruthy();
+    expect(
+      shell.test('-f', 'dist/build-withconfig.cjs.development.js')
+    ).toBeTruthy();
+    expect(
+      shell.test('-f', 'dist/build-withconfig.cjs.production.min.js')
+    ).toBeTruthy();
+    expect(
+      shell.test('-f', 'dist/build-withconfig.esm.production.min.js')
+    ).toBeTruthy();
+
+    expect(shell.test('-f', 'dist/index.d.ts')).toBeTruthy();
+
+    const lib = require(`../../${stageName}/dist`);
+    expect(lib.signature).toBe('bar 0');
+
+    expect(
+      grep(
+        'exports.signature=signature',
+        'dist/build-withconfig.cjs.production.min.js'
+      )
+    ).toBeTruthy();
```

Comment:
pattern only for simple level (not with terser, not with advanced)

Reply:
Should have a comment like there is in the advanced one.

Also should use `/pattern/` instead of `'pattern'` for explicitness because it gets evaluated like a regex and not a plain string. I tried using a plain string when I was writing some similar tests and it errored out because a regex would have required an escape there. Syntax highlighting would've shown that too.

Reply:
yep, fixed